examples/http-gateway/cloudflare/README.md

```diff
@@ -0,0 +1,19 @@
+# Cloudflare Worker based ICP HTTP Gateway
```

Comment:
nit: it seems to me "based" should be preceded with a hyphen, but am not sure whether it should be "Cloudflare-Worker-based" or "Cloudflare Worker-based" (and obviously I'm not a native speaker 😢 ).  Maybe the best solution would be to reformulate to "An ICP HTTP Gateway based on a Cloudflare Worker"?